### ISSUE TITLE
Admin UI crash when using Location field type and attempt to filter.

### ISSUE THREAD

**Issue by superthing001** [keystonejs/keystone#1874]
# Bug report

## Describe the bug

When using the Field Type `Location`, the Admin UI goes to a white screen when clicking on the filters popup. 
Looking into the console shows that `getFilterTypes` is not a function. 
Seems like the Controller is missing that function. [here](https://github.com/keystonejs/keystone/blob/master/packages/fields/src/types/Location/views/Controller.js)

## To Reproduce
- add the following to keystone's `index.js`
```
keystone.createList("Todo", {
  schemaDoc: "A list of things which need to be done",
  fields: {
    name: { type: Text, schemaDoc: "This is the thing you need to do" },
    location: { type: Location, googleMapsKey: "**key**" }
  }
});
```
- add your google maps key
- go to admin UI todos section `<baseurl>/admin/todos`
- click on the Filters dropdown button/link

## Expected behaviour

The same behaviour as normal with filters, a dropdown with available filters to select from.

To quickly fix the bug, the filter iterator can skip locations field types. However, for location filters I see two general ways to filter. 
1. a map based filter showing all items as markers on a map, receiving only items that are in bounds on the map (or some set bounds by dropdown, eg: 1mile, 20miles 100miles)
2. text based filter based on location to be typed in along with radius from that inputed location. This option would be more in line with current filter patterns.

happy to contribute,

**Comment by timleslie** [comment]
I think there are actually two bugs here. 1) The one you described but also 2) we should have a default `getFilterTypes = () => []` on the base `Controller` class.

**Comment by jordanoverbye** [comment]
Hey @superthing001 thank you for reporting this issue! We have fixed this issue in PR #1875 but I have opened another issue (#1877) to add specific filtering options to the Location Field. All PRs are welcome :)
